templates/blog/article_list.html

```diff
@@ -1,27 +1,28 @@
 {% from 'blog/macros.html' import article_list with context %}
 {% extends 'layout.html' %}
 
-{% block title %}Neomad articles {% if articles[0] %} - {{ articles[0].title|truncate(40) }}{% endif %}
+{% block title %}
+  Neomad articles
+  {% if articles[0] %} - {{ articles[0].title|truncate(40) }}{% endif %}
 {% endblock %}
-{% block description %}Read the latest Neomad's articles on Neomad.
+{% block description %}
+  Read the latest Neomad's articles on Neomad.
   {% if articles[0] %} Hotest article is « {{ articles[0].title }} ».{% endif %}
```

Comment:
`«` are French quotes. Let's use `"` in English

Reply:
qddress with 7e2287eb0d3d60e4cc531b79f1cc545f0b633aac